cmd/river/main.go

```diff
@@ -359,17 +359,33 @@ func migrateDown(ctx context.Context, logger *slog.Logger, out io.Writer, opts *
 	return true, nil
 }
 
+// Rounds a duration so that it doesn't show so much cluttered and not useful
+// precision in printf output.
+func roundDuration(duration time.Duration) time.Duration {
+	switch {
+	case duration > 1*time.Second:
+		return duration.Truncate(10 * time.Millisecond)
+	case duration < 1*time.Millisecond:
+		return duration.Truncate(10 * time.Nanosecond)
+	default:
+		return duration.Truncate(10 * time.Microsecond)
+	}
+}
```

Comment:
hah, thanks, I noticed this the other day and I'm glad you did too took the initiative to fix it!

Reply:
NP! Yeah, surprised I didn't do it before, but I guess it was just pure shipping mode.